CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ if(COMMAND cmake_policy)
     cmake_policy(SET CMP0005 NEW)
 endif(COMMAND cmake_policy)
 
+string(CONCAT DEP_BUILD_MSG
+  "\n-------------------------IMPORTANT----------------------------------\n"
+  "| Make sure that all OpenSim dependencies are built with the same  |\n"
+  "| CMAKE_BUILD_TYPE as OpenSim to avoid mysterious runtime errors.  |\n"
+  "-------------------------IMPORTANT----------------------------------\n")
+message(${DEP_BUILD_MSG})
+
```

Comment:
In CMake 3.0 and greater, it is possible to use `\` to extend a string over multiple lines. You could use that if you think it would be helpful.